metadata_validation_conversion/validation/WarningsAndAdditionalChecks.py

```diff
@@ -315,6 +315,7 @@ def check_ols(field_value, ontology_names, field_name, ontology_ids):
         print(ontology_ids)
         if 'text' in field_value and 'term' in field_value:
             term_label = list()
+            field_value['term'] = field_value['term'].replace(":", "_")
```

Comment:
no need to change original data as this has nothing to do with user's submission but rather a problem with ols